docs/docsite/rst/windows_winrm.rst

```diff
@@ -25,7 +25,7 @@ with the Ansible package, but can be installed by running the following::
 Authentication Options
 ``````````````````````
 When connecting to a Windows host, there are several different options that can be used
-when authentication with an account. The authentication type may be set on inventory 
+when authentication with an account. The authentication type may be set on inventory
```

Comment:
Did you mean to say 'authenticating' here?

Reply:
Yep it should be, will fix.